script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_1C.txt

```diff
@@ -97,7 +97,7 @@ Ah, I made her mad...
 But right now, I'm not interested in these 'what if' scenarios.
 What I really want to know is whether Arcueid can defeat that vampire in her current condition....
 <Page29>
-"I can't. Vlov is stronger than me right now.# My performance is simply lower than his, so if we fight him without a strategy, we'll get killed."
+"I can't. Vlov is stronger than me right now.# My abilities have simply dropped below his, so if we fight him without a strategy, we'll get killed."
```

Comment:
I do think performance is the right meaning rather than abilities, although it doesn't read well.
I'd straight up say strength in this case.

Reply:
Hmm, there's a `stronger` in the sentence before so I might avoid that word. Capabilities? Fitness? "I simply can't perform well enough [right now] to go head to head with him, so if we..."?

Reply:
Mm I didn't notice the 'stronger'. I think your suggestion works well.

Reply:
Mm I didn't notice the 'stronger'. I think your suggestion works well.